modules/account-lib/src/coin/sol/tokenEncodeDecode.ts

```diff
@@ -0,0 +1,216 @@
+import { AccountMeta, PublicKey, Signer, TransactionInstruction } from '@solana/web3.js';
+const splToken = require('@solana/spl-token');
+
```

Comment:
let's add a comment on why we even need to do this - mention that decoding is in alpha, but its not functional at the moment, so we're cherrypicking what we need until we have a stable release